action.yml

```diff
@@ -0,0 +1,82 @@
+name: Configure AWS Profile
+
+description: Configures an OIDC AWS Role Session as an aws config profile.
+
+
+inputs:
+  role-to-assume:
+    description: "ARN of the role to assume."
```

Comment:
PLS remove "." from all descriptions. thx

Reply:
I will do that in next pr